main.go

```diff
@@ -54,6 +55,26 @@ func main() {
 		case 4:
 			log.Infof("- %s cancelled %s", build.TriggeredWorkflow, buildURL)
 		}
+		if build.Status != 0 {
+			if cfg.SavePath != "" {
+				artifactsResponse, err := app.GetBuildArtifacts(build.Slug)
+				if err != nil {
+					log.Warnf("failed to get build artifacts, error: %s", err)
+				}
+				for _, artifactSlug := range artifactsResponse.ArtifactSlugs {
+					artifactObj, err := app.GetBuildArtifact(build.Slug, artifactSlug.ArtifactSlug)
+					if err != nil {
+						log.Warnf("failed to get build artifact, error: %s", err)
+					}
+
+					downloadErr := app.DownloadArtifact(cfg.SavePath+artifactObj.Artifact.ArtifactTitle, artifactObj.Artifact.ArtifactURL)
```

Comment:
Please avoid name duplication
```suggestion
					downloadErr := app.DownloadArtifact(cfg.SavePath + artifactObj.Artifact.Title, artifactObj.Artifact.DownloadURL)
```